earth_enterprise/rpms/opengee-common/post-uninstall.sh

```diff
@@ -22,18 +22,23 @@ BININSTALLROOTDIR="/etc/init.d"
 
 #------------------------------------------------------------------------------
 # Group names:
-GEGROUP_NAME=$(cat "$BININSTALLROOTDIR/gevars.sh" | grep GEGROUP | cut  -d'=' -f2)
-
 
 remove_users_groups()
 {
-    if [ -n "$(getent group "$GEGROUP_NAME")" ]; then
-        groupdel "$GEGROUP_NAME"
+    local GROUP_EXISTS=$(getent group "$GEGROUP")
+
+    # Add group if it does not exist:
```

Comment:
Minor: I don't think the command below is adding a group.

Reply:
Yes, I was trying to make add and delete consistent (they were originally very differently coded), and that is a copy-paste error.  But it may not matter since the post-uninstall is no longer in gradle.